config/cruisecontrol.properties

```diff
@@ -165,7 +165,13 @@ num.proposal.precompute.threads=1
 zookeeper.connect=localhost:2181/
 
 # The max number of partitions to move in/out on a given broker at a given time.
-num.concurrent.partition.movements.per.broker=10
+num.concurrent.inter.broker.partition.movements.per.broker=2
```

Comment:
Let's avoid changing the name of an existing config from `num.concurrent.partition.movements.per.broker` -> `num.concurrent.inter.broker.partition.movements.per.broker` -- i.e. this would be a breaking change to the users that we would like to avoid.